README.md

```diff
@@ -16,45 +28,75 @@ video:
 
 - Dynamic Instrumentation (No Code Change Required)
 - Robust Sampling
-- Fast Debug Symbol Parsing
-- Context Switch Tracking
-- Disassembly View
+- Wine/Proton Mixed-Callstack Profiling
+- Thread Scheduling and Dependency Tracing
+- Memory Tracing
+- GPU Driver Tracepoints (AMD Only)
+- Vulkan Debug Label and Command Buffer Tracing (AMD Only)
+- Manual Instrumentation
+- Source Code and Disassembly View
 - Remote Profiling
-- User Sessions
+- Fast Debug Symbol Parsing
 - Full Serialization of Captured Data
-- Tested on Unreal, Unity, Lumberyard, Qt, Doom3, PhysX, ...
+- ...
```

Comment:
I would remove "...".

Reply:
Are we then sure that we did not forgot an important feature?

Reply:
No, but if we did, "..." still doesn't provide any information.

Reply:
Agree. Keeping this open for now, if someone has a feature in mind that is missing.